packages/tables/frontend/sirius-components-tables/src/table/useTableColumns.tsx

```diff
@@ -13,13 +13,23 @@
 import { Selection, useSelection } from '@eclipse-sirius/sirius-components-core';
 import { MRT_ColumnDef } from 'material-react-table';
 import { useMemo } from 'react';
+import { makeStyles } from 'tss-react/mui';
 import { Cell } from '../cells/Cell';
 import { ColumnHeader } from '../columns/ColumnHeader';
 import { ResizeRowHandler } from '../rows/ResizeRowHandler';
 import { RowHeader } from '../rows/RowHeader';
 import { GQLCell, GQLLine, GQLTable } from './TableContent.types';
 import { UseTableColumnsValue } from './useTableColumns.types';
 
+const useStyles = makeStyles()((theme) => ({
+  cell: {
+    padding: `${theme.spacing(1)} ${theme.spacing(1)}`,
+    flex: '1',
+    height: '100%',
```

Comment:
The child component should not be in control of its layout in its parent component.